openhcl/virt_mshv_vtl/src/processor/tdx/mod.rs

```diff
@@ -704,6 +705,25 @@ impl BackingPrivate for TdxBacked {
             .as_ref()
             .map(|synic| synic.add_vp(params.vp_info.base.vp_index));
 
+        // Set the exception bitmap for VTL0.
+        if params.partition.intercept_debug_exceptions {
+            if cfg!(feature = "gdb") {
+                let initial_exception_bitmap = params
+                    .runner
+                    .read_vmcs32(GuestVtl::Vtl0, VmcsField::VMX_VMCS_EXCEPTION_BITMAP);
+
+                let exception_bitmap =
+                    initial_exception_bitmap | (1 << x86defs::Exception::DEBUG.0);
+
+                params.runner.write_vmcs32(
+                    GuestVtl::Vtl0,
+                    VmcsField::VMX_VMCS_EXCEPTION_BITMAP,
+                    !0,
+                    exception_bitmap,
+                );
+            }
```

Comment:
can we handle the case where this was requested and we didn't compile in support for this and return the corresponding error? similar to how it was handled in lib.rs